velox/functions/lib/TimeUtils.h

```diff
@@ -24,6 +24,13 @@ namespace facebook::velox::functions {
 namespace {
 inline constexpr int64_t kSecondsInDay = 86'400;
 inline constexpr int64_t kDaysInWeek = 7;
+const folly::F14FastMap<std::string, int8_t> kDayOfWeekNames{
```

Comment:
This file itself is quite bad actually, no anonymous space should be in header either...

Reply:
@Yuhta The anonymous namespace has been modified, and the F14FastMap has been moved to the cpp file.